conf/messages.en

```diff
@@ -542,8 +542,10 @@ view.amend-case-confirmation.save.description=You’ll need to use this case ref
 view.amend-case-confirmation.save.description2=It is the same case reference number that you were given when the case was first created.
 view.amend-case-confirmation.next.heading=Next steps
 view.amend-case-confirmation.next.description=Once the information checks are complete, you’ll hear from us through CHIEF or your declaration software. We’ll let you know if we need any further information from you.
-view.amend-case-confirmation.continue=Send documents for a customs check
-view.amend-case-confirmation.footnote=View guidance on GOV.UK about <a class="govuk-link" href="#">Clearing goods entering, leaving or transiting the UK or EU</a> or <a class="govuk-link" href="#">C1601, C1602, C1603 forms</a>.
+view.amend-case-confirmation.next.description2=If you need to, you can <a class="govuk-link" href="{0}">send documents for another customs check</a>.
+view.amend-case-confirmation.info.heading=If we need more information
+view.amend-case-confirmation.info.description=We’ll contact you through CHIEF or your declaration software if we need more documents from you.
```

Comment:
Duplication from the new case message keys - seemed slightly 'cleaner' to have it this way but happy to re-use